web-app/js/jquery/jquery.fileDownload.js

```diff
@@ -327,6 +327,12 @@ $.extend({
         setTimeout(checkFileDownloadComplete, settings.checkInterval);
 
 
+        function deleteCookie(name, path, domain) {
```

Comment:
There is a new `jQuery File Download Plugin v1.4.5` does it do what you want?
It seems a little wrong to mod this plugin

Reply:
I had a look at it and it doesn't do what we want. Personally, and this might just be my opinion but if we're adding code directly to our repo in this manner instead of pulling it in from it's own source then we're basically saying 'this is now part of our code' and therefore it can be editable just like the rest of our code. Otherwise we should just be managing it properly as a third-party dependency.

Reply:
Not necessarily. "Vendoring" third party libraries into your codebase is very common, but doesn't automatically make them fair game for forking. It just makes upgrading jquery a nightmare in future! There has to be a way to achieve this without modifying 3rd party libs.

Reply:
um, not really sure I know what the answer is to that. Maybe submit a PR and get the feature added to the library?

Reply:
Would it be possible to do something like the overrides we have in the prototypes folder? https://github.com/aodn/aodn-portal/tree/master/web-app/js/portal/prototypes

Reply:
thanks @akashisama I'll see if that would work